imagersite/imager_profile/admin.py

```diff
@@ -0,0 +1,13 @@
+from django.contrib import admin
+
+from .models import ImagerProfile
+
+
+# class ImagerProfileAdmin(admin.ModelAdmin):
+#     fieldsets = [
+#         (None,               {'fields': ['user']}),
```

Comment:
What you were looking for here is [Admin Inlines](https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#inlinemodeladmin-objects).  They allow you to edit the fields of a related model inline with the fields of the current model.